apps/passport-server/src/services/telegramService.ts

```diff
@@ -595,6 +595,7 @@ export class TelegramService {
     }
 
     if (
+      // TODO: wrap in a MultiProcessService?
```

Comment:
Worth measuring how expensive this is (via tracing), vs. the overhead of the MultiProcess service.

Reply:
I'll follow up later - tracing is in place to measure this